db/version_set.cc

```diff
@@ -3834,10 +3833,7 @@ Status VersionSet::ReduceNumberOfLevels(const std::string& dbname,
   ColumnFamilyDescriptor dummy_descriptor(kDefaultColumnFamilyName,
                                           ColumnFamilyOptions(*options));
   dummy.push_back(dummy_descriptor);
-  {
-    InstrumentedMutexLock l(&dummy_mutex);
```

Comment:
What's the change for?

Reply:
to revert some old change that is no longer needed